src/variants/model/bias/mod.rs

```diff
@@ -74,27 +74,35 @@ impl BiasesBuilder {
 impl Biases {
     pub(crate) fn all_artifact_combinations(
         consider_read_orientation_bias: bool,
-    ) -> Box<dyn Iterator<Item = Self>> {
-        if consider_read_orientation_bias {
-            Box::new(
+        consider_strand_bias: bool,
+    ) -> Option<Box<dyn Iterator<Item = Self>>> {
```

Comment:
When using [`std::iter::empty()`](file:///home/johannes/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/share/doc/rust/html/std/iter/fn.empty.html) for the case where both args are false, you can avoid the Option type.

Reply:
Ah, cool. I was looking for something like this, but didn't know how and where to look. A classical Rust thing...

Reply:
Done.